From 3b9c294e579b33ff2218fedba3b7e20e6b291a34 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@freescale.com>
Date: Thu, 29 Oct 2015 17:07:32 +0800
Subject: [PATCH] MLK-11790 imx_v7_defconfig: remove atheros wifi support

make savedefconfig shows CONFIG_KEYBOARD_SNVS_PWRKEY and
CONFIG_MXC_MLB150 also needs to be removed.
So remove them together.

Signed-off-by: Dong Aisheng <aisheng.dong@freescale.com>
---
 arch/arm/configs/imx_v7_defconfig | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index abdf1df..e3315f3 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -152,18 +152,15 @@ CONFIG_USB_RTL8150=m
 CONFIG_USB_RTL8152=m
 CONFIG_USB_USBNET=m
 CONFIG_USB_NET_CDC_EEM=m
-CONFIG_ATH_CARDS=y
-CONFIG_ATH6KL=m
-CONFIG_ATH6KL_SDIO=m
 CONFIG_BCMDHD=m
 CONFIG_BCMDHD_SDIO=y
 CONFIG_BCMDHD_FW_PATH="/lib/firmware/bcm/fw_bcmdhd.bin"
 CONFIG_BCMDHD_NVRAM_PATH="/lib/firmware/bcm/bcmdhd.cal"
+# CONFIG_RTL_CARDS is not set
 # CONFIG_INPUT_MOUSEDEV_PSAUX is not set
 CONFIG_INPUT_EVDEV=y
 CONFIG_INPUT_EVBUG=m
 CONFIG_KEYBOARD_GPIO=y
-CONFIG_KEYBOARD_SNVS_PWRKEY=y
 CONFIG_KEYBOARD_IMX=y
 CONFIG_MOUSE_PS2=m
 CONFIG_MOUSE_PS2_ELANTECH=y
@@ -341,7 +338,6 @@ CONFIG_MXC_IPU=y
 CONFIG_MXC_GPU_VIV=y
 CONFIG_MXC_IPU_V3_PRE=y
 CONFIG_MXC_MIPI_CSI2=y
-CONFIG_MXC_MLB150=m
 CONFIG_MXC_SIM=y
 CONFIG_NEW_LEDS=y
 CONFIG_LEDS_CLASS=y
-- 
1.8.0

